level_creator.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/python
+
+from enum import Enum
+
+import sys
+import json
+
+class StarType(Enum):
+    BRONZE = 'bronze'
+    SILVER = 'silver'
+    GOLD = 'gold'
+
+usable_colors = []
+
+
+
+#### CONSOLE READER HELPER FUNCTIONS ####
+def parse_int_input_with_bounds(prompt: str, min: int, max: int):
```

Comment:
Do I need to know what this is, I have no clue

Reply:
lol it's a script to write levels easily without manually writing json files